### PR TITLE
Handling the multiple cmd del call from kubelet

### PR DESCRIPTION
**Pull request by rkamudhan**
Refer #115 for more information

### CODE REVIEW

**Comment by rkamudhan** [comment]
@dcbw @s1061123 @dougbtv  Added fix with reference to https://github.com/kubernetes/kubernetes/issues/43014#issuecomment-287164444

File: multus/multus.go [hunk 1]
```diff
@@ -294,6 +294,24 @@ func cmdDel(args *skel.CmdArgs, exec invoke.Exec, kubeClient k8s.KubeClient) err
 		return err
 	}
 
+	if args.Netns == "" {
+		return nil
+	}
+	netns, err := ns.GetNS(args.Netns)
+	if err != nil {
+		if err != nil {
+			//  if NetNs is passed down by the CEO, or if it called multiple times
```
Comment (s1061123) [inline multus/multus.go]: one q. what is the CEO?

File: multus/multus.go [hunk 2]
```diff
@@ -294,6 +294,24 @@ func cmdDel(args *skel.CmdArgs, exec invoke.Exec, kubeClient k8s.KubeClient) err
 		return err
 	}
 
+	if args.Netns == "" {
+		return nil
+	}
+	netns, err := ns.GetNS(args.Netns)
+	if err != nil {
+		if err != nil {
```
Comment (s1061123) [inline multus/multus.go]: "if err != nil {" is duplicated and nested. How about to make it one?

File: multus/multus.go (hunk 1 above)
Comment (rkamudhan) [reply multus/multus.go]: Sorry that should Cloud Orchestration Engine

File: multus/multus.go (hunk 2 above)
Comment (rkamudhan) [reply multus/multus.go]: I don't know how it came it.. My bad

**Comment by rkamudhan** [comment]
@s1061123 Sorry, I didn't get back to you on this. I will take your feedback and update the code today. I am target it for the [v3.1](https://github.com/intel/multus-cni/milestone/4)

**Pull request merged**